### DISCUSSION TITLE
Sometimes I dont get the reason for pos_tags identification.

### DISCUSSION THREAD

**Discussion by Emporea** [explosion/spaCy#10156] [Help: Model Advice]
Using en_core_web_sm or en_core_web_lg makes no difference.

```
[
  ['v3.9.0', 'ADJ'],
  ['v3.9.1', 'PROPN'],
  ['v3.9.2', 'X'],
  ['v3.9.3', 'ADP']
]
```

How is it possible that these 4 tokens get so different pos_tags?

**Comment by polm** [comment]
I'm not sure why those are so different, but the issue is related to the fact that the model hasn't seen any tokens like that during training. See #3052 - debugging individual results is generally hard.
